packages/now-cli/src/util/get-frameworks.ts

```diff
@@ -2,5 +2,7 @@ import { Framework } from '@now/frameworks';
 import Client from './client';
 
 export async function getFrameworks(client: Client) {
-  return await client.fetch<Framework[]>('/v1/frameworks');
+  return await client.fetch<Framework[]>('/v1/frameworks', {
+    useCurrentTeam: false,
+  });
```

Comment:
I added this here because the `?teamId` query string param doesn't change the response, the frameworks are always the same.